pkg/operation/botanist/controlplane.go

```diff
@@ -515,16 +515,48 @@ func (b *Botanist) DeploySeedLogging() error {
 	}
 
 	var (
-		credentials = b.Secrets["logging-ingress-credentials"]
-		basicAuth   = utils.CreateSHA1Secret(credentials.Data[secrets.DataKeyUserName], credentials.Data[secrets.DataKeyPassword])
-		kibanaHost  = b.Seed.GetIngressFQDN("k", b.Shoot.Info.Name, b.Garden.Project.Name)
+		kibanaHost                              = b.Seed.GetIngressFQDN("k", b.Shoot.Info.Name, b.Garden.Project.Name)
+		kibanaCredentials                       = b.Secrets["kibana-logging-sg-credentials"]
+		kibanaAdminIngressCredentialsSecresName = "logging-ingress-credentials"
+		kibanaUserIngressCredentialsSecresName  = "logging-ingress-user-credentials"
```

Comment:
To harmonize it with the monitoring PR: please rename to `logging-ingress-credentials-users` (everywhere)